mappings/net/minecraft/util/crash/CrashReport.mapping

```diff
@@ -12,13 +12,15 @@ CLASS net/minecraft/class_128 net/minecraft/util/crash/CrashReport
 	METHOD method_555 addStackTrace (Ljava/lang/StringBuilder;)V
 	METHOD method_556 addElement (Ljava/lang/String;I)Lnet/minecraft/class_129;
 		ARG 1 name
+		ARG 2 ignoredStackTraceCallCount
```

Comment:
I believe `depthOffset` or `skippedDepth` are better alternatives to this current name.

Reply:
There should be a "stack trace" in the name. The method just adds a section to the crash. It's not clear that the parameter is talking about the trace.